arch/arm/boot/dts/bcm2711-rpi-cm4.dts

```diff
@@ -65,6 +65,18 @@
 		enable-active-high;
 		gpio = <&expgpio 6 GPIO_ACTIVE_HIGH>;
 	};
+
+	cam1_reg: cam1_reg {
+		compatible = "regulator-fixed";
+		regulator-name = "cam1-reg";
+		gpio = <&expgpio 5 GPIO_ACTIVE_HIGH>;
+		enable-active-high;
+		status = "disabled";
+	};
+
+	aliases {
+		cam0_reg = &cam1_reg;
```

Comment:
You'll have to explain this, especially as it's the only cam_reg alias.

Reply:
This is probably where I've got it wrong between aliases and labels.

CM4 shares the same GPIO to control both CAM0 & CAM1, so I want the same regulator node to be referenced as either cam1_reg or cam0_reg from overlays.

Reply:
Internal references usually require phandles, which are the compiled equivalent of labels. Just put a second, `cam0_reg` label on the `cam1_reg` node.